app/models/resource.rb

```diff
@@ -1,6 +1,6 @@
 class Resource < ApplicationRecord
   include Elasticsearch::Model
-  index_name "articles-#{Rails.env}"
+  index_name "#{self.class.name.downcase}-#{Rails.env}"
```

Comment:
Good stuff. We could pluralize it to follow the previous style. I'm also in the habit of making logic "paths" for caches/indexes as in using `/development/resources`, but that's just a  personal preference - totally ok with using model-env and I will use the same format if I need to in the rest of the project.